django/db/backends/base/schema.py

```diff
@@ -838,6 +838,17 @@ def alter_field(self, model, old_field, new_field, strict=False):
             strict,
         )
 
+    def _field_db_check(self, field, field_db_params):
+        # Check constraints always with the same column name to avoid
+        # recreating constrains when the column is renamed.
```

Comment:
`always have the same column name` ?

Reply:
We want to check CHECK constraints SQL using always the same mocked column name (`__column_name__`) to avoid recreating constraints when the column is renamed. For example:
```
>> old_db_params["check"]
"column" > 0
>> new_db_params["check"]
"renamed_column" > 0
```
with this patch compared statements are the same and we avoid `DROP CONSTRAINT` and `ADD CONSTRAINT`.

I know that this comment is not clear enough :disappointed: